### PR TITLE
Use version 0.2.0

### PR DESCRIPTION
**Pull request by jitseniesen**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jitseniesen** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ requirements:
   run:
     - memory_profiler
     - python
-    - spyder >=3
+    - spyder >=4
```
Comment (ccordoba12) [inline recipe/meta.yaml:27]: I think we should limit this to

    - spyder >=4,<5

@jitseniesen, what do you think?

Comment (jitseniesen) [reply recipe/meta.yaml:27]: I like your idea, I will do that in the future

Comment (ccordoba12) [reply recipe/meta.yaml:27]: Ok, thanks!

**Pull request merged**